packages/apps/job-launcher/client/src/constants/chains.ts

```diff
@@ -4,20 +4,29 @@ export const IS_MAINNET =
   import.meta.env.VITE_APP_ENVIRONMENT.toLowerCase() === 'mainnet';
 export const IS_TESTNET = !IS_MAINNET;
 
-export const SUPPORTED_CHAIN_IDS = IS_MAINNET
-  ? [ChainId.POLYGON] // ? [ChainId.BSC_MAINNET, ChainId.POLYGON, ChainId.MOONBEAM]
-  : [
+export let SUPPORTED_CHAIN_IDS: ChainId[];
+switch (import.meta.env.VITE_APP_ENVIRONMENT.toLowerCase()) {
+  case 'mainnet':
+    SUPPORTED_CHAIN_IDS = [ChainId.POLYGON];
+    break;
+  case 'testnet':
+    SUPPORTED_CHAIN_IDS = [
       ChainId.BSC_TESTNET,
       ChainId.POLYGON_MUMBAI,
       ChainId.GOERLI,
-      ChainId.X1_TESTNET,
```

Comment:
You are removing what you added in this PR

Reply:
have undo this